javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const array = [];
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      delete null;
```

Comment:
`delete` is only used to delete a property from an object, so it shouldn't be used here.